packages/echo/src/client.ts

```diff
@@ -339,30 +339,6 @@ export class Echo {
     return headers;
   }
 
-  public async diff(echoUrl: string, anonymous?: string): Promise<unknown> {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Yay for code removal!‏